pyproject.toml

```diff
@@ -61,7 +61,7 @@ dev = [
 path = "src/RepoAuditor/__init__.py"
 
 [tool.pytest.ini_options]
-addopts = "--verbose -vv --capture=no --cov=RepoAuditor --cov-fail-under=95.0"
+addopts = "--verbose -vv --capture=no --cov=RepoAuditor --cov-fail-under=95.0 --cov-report html"
```

Comment:
If we add one coverage report style, we need to add them all: `--cov-report html --cov-report term --cov-report xml:coverage.xml`. (I'm surprised that `term` and `xml` weren't already there).

Also make sure that we are ignorning `coverage.xml` and the output html file/directory.

Reply:
You got it